components/status_dropdown/status_dropdown.tsx

```diff
@@ -290,21 +300,20 @@ export default class StatusDropdown extends React.PureComponent {
             id: 'dndSubMenu-header',
             direction: 'right',
             text: localizeMessage('status_dropdown.dnd_sub_menu_header', 'Disable notifications until:'),
-        }].concat(
+        } as any].concat(
             this.dndTimes.map((time, index) => {
                 return {
                     id: `dndTime-${time.split(' ').join('')}`,
                     direction: 'right',
                     text: localizeMessage('status_dropdown.dnd_sub_menu_item.time', time),
                     action: index === 4 ? () => setCustomTimedDnd() : () => setDnd(event, index),
```

Comment:
Hi @devinbinnie , for event data type, what  is the best data type for this ?
Currently this code will return eslint error "'event' is deprecated".

Reply:
@arvinDarmawan I don't think that's a typing issue but I think we can ignore that issue for now since it won't trip up the type checker.

Reply:
It will return error on type-check @devinbinnie 
<img width="1407" alt="Screen Shot 2021-05-18 at 09 13 49" src="https://user-images.githubusercontent.com/28821618/118580095-771aa600-b7b9-11eb-80f7-e775bf7c6727.png">

Reply:
I wonder if we can just remove references to `event`? I'm not even sure where the reference to `event` is coming from, so it's probably always going to be undefined.

Reply:
Hi @devinbinnie , done.